twitter/Models/Tweet.h

```diff
@@ -0,0 +1,33 @@
+//
+//  Tweet.h
+//  twitter
+//
+//  Created by Joey R. Hernandez Perez on 6/29/20.
+//  Copyright © 2020 Emerson Malca. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import "User.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface Tweet : NSObject
+
+@property (nonatomic, strong) NSString *idStr; // For favoriting, retweeting & replying
+@property (nonatomic, strong) NSString *text; // Text content of tweet
+@property (nonatomic) int favoriteCount; // Update favorite count label
```

Comment:
This is a bit of a dangerous way to think. You're talking about labels but consider if you were building an api built on twitter's api. Maybe you pull favorite counts from different places and return them in the same payload. This is why you don't want to tie your model to the view even in comments.